src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,26 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {`Posted by `}
+        <UserInfo user={post.user} />
```

Comment:
user can be null, so let's render this component only when a user exists
```suggestion
        {post.user && <UserInfo user={post.user} />}
```